v8py/debug.py

```diff
@@ -1,8 +1,13 @@
 import _v8py
 
+
+class PlainDebugger(_v8py.Debugger):
+    pass
+
+
 class Debugger(_v8py.Debugger):
     def __init__(self, context):
-        super(self.__class__, self).__init__(context)
+        _v8py.Debugger.__init__(self, context)
```

Comment:
Oh, snap, accidentally also committed also the debugger fix. Inheriting from `Debugger` caused stack overflow